CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ _None_
 
 ### New Features
 
-_None_
+* Add a new `custom` command
```

Comment:
Use the same format as the other entries, namely:
- End your sentence with a `.` and 2 spaces
- Author (you have this)
- PR

Reply:
Alright, thanks.